src/util/listQueryBuilder.js

```diff
@@ -471,7 +580,24 @@ function getSqlExpression (attr) {
   return `"${column}"#>>'{${parts.slice(1).join(',')}}'`
 }
 
+function getMinRangeValue (rangeOrValue) {
+  if (_.isUndefined(rangeOrValue)) return
+
+  const isSingleValue = !_.isPlainObject(rangeOrValue)
+  if (isSingleValue) return rangeOrValue
+
+  const rangeValues = [rangeOrValue.gt, rangeOrValue.gte]
+  const [minValue] = rangeValues.sort((a, b) => {
+    if (_.isUndefined(a)) return 1
+    else if (_.isUndefined(b)) return -1
+    else return a < b ? -1 : 1
+  })
```

Comment:
Very contrived, you can make this much simpler to read (e.g without sort function).